bigbluebutton-html5/imports/ui/components/video-provider/hooks/index.ts

```diff
@@ -192,10 +192,7 @@ export const useDisableCam = () => {
   return meeting?.lockSettings ? meeting?.lockSettings.disableCam : false;
 };
 
-const getCountData = () => {
-  const { data: countData } = useDeduplicatedSubscription<UsersCountSubscriptionResponse>(
-    USER_AGGREGATE_COUNT_SUBSCRIPTION,
-  );
+const getCountData = (countData) => {
```

Comment:
I think a better approach would be to prefix this function’s name with `use`, since it’s actually a hook, and leave the users count subscription hook call inside it. The linter error persists only because hook names are expected to start with use. However, if you decide to keep the current approach, make sure to add explicit types to the `countData` parameter to avoid the implicit any error.
```suggestion
const useCountData = () => {
```